chainercv/datasets/online_products/online_products_dataset.py

```diff
@@ -49,20 +49,20 @@ class OnlineProductsDataset(chainer.dataset.DatasetMixin):
         data_dir (string): Path to the root of the training data. If this is
             :obj:`auto`, this class will automatically download data for you
             under :obj:`$CHAINER_DATASET_ROOT/pfnet/chainercv/online_products`.
-        mode ({'train', 'test'}): Mode of the dataset.
+        split ({'train', 'test'}): split of the dataset.
```

Comment:
How about `Select a split of the dataset`? Anyway, the sentence should start with upper case.